app/src/App/DesktopApp.tsx

```diff
@@ -141,6 +141,10 @@ function RobotControlTakeover(): JSX.Element | null {
   const params = deviceRouteMatch?.params as DesktopRouteParams
   const robotName = params?.robotName
   const robot = useRobot(robotName)
+  const isOT3 = useIsOT3(robotName)
+
+  // E-stop is not supported on OT2
+  if (isOT3) return null
```

Comment:
don't we want this to be the opposite? 

```suggestion
  if (!isOT3) return null
```

Reply:
oooooooh, thanks.
You are absolutely right!